packages/terra-tag/package.json

```
@@ -23,14 +23,18 @@
   "homepage": "https://github.com/cerner/terra-core#readme",
   "peerDependencies": {
     "react": "^16.8.5",
-    "react-dom": "^16.8.5"
+    "react-dom": "^16.8.5",
+    "react-intl": ">=2.8.0 <6.0.0"
```

For better readability
```suggestion
    "react-intl": "^2.8.0 || 3 || 4 || 5"
```